orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -115,7 +115,8 @@ def load_features(self):
         self.selected_features = []
         if self.corpus is not None:
             self.features = [(meta.name, vartype(meta)) for meta in self.corpus.domain.metas] + \
-                            [(attr.name, vartype(attr)) for attr in self.corpus.domain.attributes]
+                            [(attr.name, vartype(attr)) for attr in self.corpus.domain.attributes] + \
+                            [(class_var.name, vartype(class_var)) for class_var in self.corpus.domain.class_vars]
```

Comment:
I can't argue with your argument that iterator must iterate over the entire contents. You won: if we have `Domain.__iter__`, it must include metas.

It still contradicts what I would expect from it. If the confusion is real and not just mine, it is better to have no `__iter__`. But it's quite possibly just my bias, so submit a PR and let (at least) @markotoplak and @astaric vote about it. I'll abstain. :)